### PR TITLE
added isProfiling() to ReactDebugTool and isRunning() to PerfTools

### LINKED ISSUE #6762: ReactPerf 15.1.0-alpha.1 expose isProfiling on the exported object?
I've tested the new Perf tools on one of the screens in one app I'm working on and it works great :)

One thing I wanted also on the old one Perf tools was to be able to check whether the Perf tools are started or stopped, so basically exposing `ReactDebugTool`'s `isProfiling` variable. My use for that is that I have a keyboard shortcut (in development) bound to start/stop the Perf tools and currently I'm forced to maintain such a variable myself to know whether to start or stop (which gets out of sync if I start/stop the perf tools without the keyboard shortcut).

If you don't think it's worthwhile to expose `isProfiling` feel free to close this issue.

### PR DESCRIPTION
**Pull request by nfcampos**
I'm not sure the tests are added are worth it, 

closes #6762

### CODE REVIEW

**Comment by nfcampos** [comment]
should I add `isRunning` to the docs? is it this file https://github.com/facebook/react/blob/master/docs/docs/10.9-perf.md?

**Comment by nfcampos** [comment]
the build timed out for some reason, not really sure why

File: src/isomorphic/__tests__/ReactPerf-test.js [hunk 1]
```diff
@@ -283,4 +285,12 @@ describe('ReactPerf', function() {
     ReactPerf.printDOM(measurements);
     expect(console.error.calls.length).toBe(1);
   });
+
+  it('returns isRunning state', () => {
```
Comment (gaearon) [inline src/isomorphic/__tests__/ReactPerf-test.js]: It would be better to test this by running `start()` and `stop()`.

Comment (nfcampos) [inline src/isomorphic/__tests__/ReactPerf-test.js]: I've changed it

**Comment by facebook-github-bot-0** [comment]
@nfcampos updated the pull request.

**Comment by gaearon** [comment]
Let’s add extra tests that verify that running `begin`/`start` or `end`/`stop` twice has no effect.

**Comment by nfcampos** [comment]
separate tests right? because they're testing `start` etc. instead of `isRunning`

**Comment by gaearon** [comment]
>separate tests right? because they're testing start etc. instead of isRunning

Yeah. I’d probably just test the public API here (`ReactPerf`). Basically to verify that it doesn’t crash when you try to `start()` or `stop()` it twice, and that `isRunning()` returns a sensible value.

**Comment by nfcampos** [comment]
@gaearon I've added those two tests to `ReactPerf`

**Comment by facebook-github-bot-5** [comment]
@nfcampos updated the pull request.

**Comment by gaearon** [comment]
Thanks!

**Pull request merged**